src/m365/pp/commands/gateway/gateway-list.ts

```diff
@@ -0,0 +1,42 @@
+import { Logger } from '../../../../cli';
+import request from '../../../../request';
+import PowerBICommand from '../../../base/PowerBICommand';
+import commands from '../../commands';
+import { Gateway } from '../../Gateway';
+
+class PpGatewayListCommand extends PowerBICommand {
+  public get name(): string {
+    return commands.GATEWAY_LIST;
+  }
+
+  public get description(): string {
+    return 'Returns a list of gateways for which the user is an admin';
+  }
+
+  public defaultProperties(): string[] | undefined {
+    return ['id', 'name'];
+  }
+
+  public commandAction(logger: Logger, args: any, cb: () => void): void {
+    if (this.verbose) {
+      logger.logToStderr(`Retrieving list of gateways for which the user is an admin...`);
+    }
+
+    const requestOptions: any = {
+      url: `${this.resource}/v1.0/myorg/gateways`,
+      headers: {
+        accept: 'application/json'
+      },
+      responseType: 'json'
+    };
+
+    request
+      .get<{ value: Gateway[] }>(requestOptions)
+      .then((res: { value: Gateway[] }): void => {
```

Comment:
Since we're not doing anything with the type of the retrieved data, we might as well simplify it and define the return type as `{ value: any[] }`